src/scss/misc/_colors.scss

```diff
@@ -18,7 +18,7 @@ $colors: (
   assets: (
     oil: (color: #729e00, icon: 'fa-database'),
     gas: (color: #ff7a28, icon: 'fa-fire'),
-    recycle: (color: #f1b71c, icon: 'fa-recycle'),
+    recycle: (color: #b11f24, icon: 'fa-recycle'),
```

Comment:
The problem i have is this wont be a short term. What will end up happening is well put this up and use this and wont take the time to go back and actually update the colors to what they should be. Case in point Coy has had the task of updating our colors for a very long time. and the answer always is "we dont have time". We cant keep taking shortcuts for everything we do it is adding too much debt and we are getting crushed by it. Have coy finish that up and lets roll out what the colors should be.